process-engine/bpmn-studio
# ⬆️ Update Backend Dependencies

**Changes:**

1. Update backend dependencies
2. Fix merging tokens of parallel gateways

**Issues:**

Related to https://github.com/process-engine/process_engine_runtime/issues/152

PR: #1135

## Example

Running [this](https://github.com/process-engine/bpmn-studio/files/2604336/ParallelTest.bpmn.zip) process:
![paralleltest](https://user-images.githubusercontent.com/20394992/48844122-40ef3d80-ed99-11e8-87e7-9f71e7d90318.png)

would lead to this token in the `confirm`-UserTask

<img width="979" alt="screenshot 2018-11-21 at 14 22 20" src="https://user-images.githubusercontent.com/20394992/48844020-0f767200-ed99-11e8-9dea-fdba3251534d.png">



## How can others test the changes?

- Run [this](https://github.com/process-engine/bpmn-studio/files/2604336/ParallelTest.bpmn.zip) process, or any other process with a parallel gateway
- Go into the `inspect correlation`-view
- **Notice that the merging of the tokens should now work.**

## PR-Checklist

Please check the boxes in this list after submitting your PR:


## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️     |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️     |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>
